### PR TITLE
Task/caps lock warning

### LINKED ISSUE #886: Sitewide - Caps Lock warning when entering password
# Description
Add a Caps Lock warning for when users accidentally have it on, when typing in passwords 
Example: https://www.w3schools.com/howto/howto_js_detect_capslock.asp

- [ ] This task requires a unit test.

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #886 

### Notes:
- Made a caps lock warning during password input

### CODE REVIEW

File: resources/views/auth/login.html.twig [hunk 1]
```diff
@@ -95,6 +95,7 @@
                                 required
                                 type="password"
                                 autocomplete="off" />
+                            <span class="password-caps-warning hidden">Warning: Caps lock engaged</span>
```
Comment (gggrant) [inline resources/views/auth/login.html.twig]: This entry should be in a lang file with a French equivalent.

File: resources/views/auth/register.html.twig [hunk 2]
```diff
@@ -151,11 +150,9 @@
                                         type="password"
                                         autocomplete="off" />
                                 </div>
-
                             </div>
-
                         </div>
-
+                        <span class="password-caps-warning hidden">Warning: Caps lock engaged</span>
```
Comment (gggrant) [inline resources/views/auth/register.html.twig]: My password manager adds a div underneath the password field and obscures the text. Would above be better for most cases? Not a big deal.

**Comment by yonikid15** [comment]
@gggrant are you sure it was working for me. It should appear when the password is in focus, the user is typing AND the caps lock is on. When the password input loses focus it will be set to hidden.

File: resources/views/auth/login.html.twig (hunk 1 above)
Comment (yonikid15) [reply resources/views/auth/login.html.twig]: Damn missed that sorry

**Comment by yonikid15** [comment]
This is getting added to clone-framework

**Pull request closed without merging**